start/sw/web-app/laravel5.md

```diff
@@ -29,6 +29,13 @@ OR, you can rip the benefit of using Fedora, and install composer directly inste
 $ sudo dnf install composer
 ```
 
+[Apache](/start/sw/web-app/apache.html) also needs the `php` package to work correctly.
+```bash
+# If you are using Apache as your http server:
+$ sudo dnf install php
```

Comment:
Please drop the PHP install line, as php installation is already mentined above.